src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -295,13 +294,20 @@ export default class BrowseSkill extends React.Component {
                                 </div>
                                 </Link>
                                 <div style={styles.rating}>
-                                    <span style={styles.positive}>
-                                        <Like style={styles.like} />
-                                        {positive_rating}
-                                    </span>
-                                    <span style={styles.negative}>
-                                        <Dislike style={styles.dislike} />
-                                        {negative_rating}
+                                    <Ratings
+                                        rating={average_rating || 2.4}
```

Comment:
Let's use 0 as a default instead. :thinking:

Reply:
@anshumanv Yes, I think there was some error while committing. Making the change right away!

Reply:
Cool thanks! :smile: